meshers/transvoxel/transvoxel.h

```diff
@@ -9,6 +9,7 @@
 
 #include <core/math/color.h>
 #include <vector>
+#include <stddef.h>
```

Comment:
Was it required here? I think `voxel_buffer_internal.h` already includes `Span`

Reply:
Looks you are right, it still compiles without that import :)
Updated the pr